src/main/java/io/github/cottonmc/cotton/gui/widget/WButton.java

```diff
@@ -99,14 +101,14 @@ public void paint(MatrixStack matrices, int x, int y, int mouseX, int mouseY) {
 		float buttonWidth = halfWidth*px;
 		float buttonHeight = 20*px;
 		
-		float buttonEndLeft = (200-(getWidth()/2)) * px;
+		float buttonEndLeft = (200-((float)getWidth()/2)) * px;
```

Comment:
The old int division is correct, floats can cause weird rendering that doesn't align with the grid

Reply:
Alright haha